### DISCUSSION TITLE
Change locale and geolocation of page?

### DISCUSSION THREAD

**Discussion by xao0isb** [rubycdp/ferrum#484] [General]
Is it possible? Or I need to use some other solutions?

**Comment by xao0isb** [comment]
I know that I can change geolocation using the proxy. But what about locale?

**Comment by route** [comment]
There’s even rspec test in the lib https://github.com/rubycdp/ferrum/blob/5ca5e9ed5e9ac8be580589e234aadc1de6929bc0/spec/network_spec.rb#L388
And this is how it’s done in JS way: https://stackoverflow.com/questions/46713871/setting-sensors-location-in-headless-chrome

**Comment by route** [comment]
Try --lang=en

**Reply by xao0isb** [reply]
Where should I pass it?

**Reply by route** [reply]
https://github.com/rubycdp/ferrum?tab=readme-ov-file#customization

In browser options `Ferrum::Browser.new(browser_options: { lang: "de" })`

**Reply by xao0isb** [reply]
Oh thanks!
